tests/imposter/README.md

```diff
@@ -6,16 +6,16 @@
 
 Imposter can be [run locally](https://docs.imposter.sh/getting_started/) using the [Imposter CLI](https://docs.imposter.sh/run_imposter_cli/). Alternatively, it can also be run using [Docker](https://docs.imposter.sh/run_imposter_docker/) or a [JAR](https://docs.imposter.sh/run_imposter_jar/) file.
 
-The Imposter server can be started using the cli:
+The Imposter server can be started using the cli (this will use the latest version of imposter):
```

Comment:
If you pass `-v VERSION` you can set the version to use, e.g.

```shell
imposter up -v 4.1.2
```

Or, if you want to pin it using a config file you can create a `.imposter.yaml` file and set the version there:

```yaml
# .imposter.yaml
---
version: "4.1.2"
```